app/Http/Requests/EditUserRequest.php

```diff
@@ -35,4 +35,29 @@ public function rules()
             'address' => 'required|min:6',
         ];
     }
+
+    public function messages()
+    {
+        return [
+            'email.required' => __('validation.required', ['attribute' => __('admin.form.email')]),
+            'fullname.required' => __('validation.required', ['attribute' => __('admin.form.fullname')]),
+            'username.required' => __('validation.required', ['attribute' => __('admin.form.username')]),
+            'username.required' => __('validation.required', ['attribute' => __('admin.form.username')]),
+            'phone.required' => __('validation.required', ['attribute' => __('admin.form.phone')]),
+            'birthday.required' => __('validation.required', ['attribute' => __('admin.form.birthday')]),
+            'gender.required' => __('validation.required', ['attribute' => __('admin.form.gender')]),
+            'role_id.required' => __('validation.required', ['attribute' => __('admin.form.role_id')]),
+            'address.required' => __('validation.required', ['attribute' => __('admin.form.address')]),
+            'birhday.date' => __('validation.date', ['attribute' => __('admin.form.birthday')]),
+            'phone.numeric' =>  __('validation.only_num', ['attribute' => __('admin.form.phone')]),
```

Comment:
**Reporter**: CHECKSTYLE</br>**Rule**: Framgia.Arrays.ArrayDeclaration.SpaceAfterDoubleArrow</br>**Severity**: error</br>**File**: app/Http/Requests/EditUserRequest.php</br>**Line**: 52</br>Expected 1 space between double arrow and "__"; 2 found